src/Factory.php

```diff
@@ -19,12 +19,13 @@ public static function create(
         LoopInterface $loop,
         array $options = []
     ): Client {
+        $container = self::createContainer($loop, $options);
         return new Client(
```

Comment:
Do we actually need this method? This factory should return the commandbus, not the client?

Reply:
Ignore this comment for now. It would be something for a follow up PR, like to explore the idea though.